vault/resource_pki_secret_backend_root_sign_intermediate.go

```diff
@@ -324,12 +353,15 @@ func pkiSecretBackendRootSignIntermediateCreate(ctx context.Context, d *schema.R
 		consts.FieldStreetAddress,
 		consts.FieldPostalCode,
 		consts.FieldSignatureBits,
+		consts.FieldSKID,
 		consts.FieldNotAfter,
+		consts.FieldNotBeforeDuration,
 	}
 
 	intermediateSignBooleanAPIFields := []string{
 		consts.FieldExcludeCNFromSans,
 		consts.FieldUseCSRValues,
+		consts.FieldUsePSS,
```

Comment:
Since this was added in 1.12, does it need a version check?

Reply:
Good catch, I have added a skip function.